src/model/consent/consent.ts

```diff
@@ -0,0 +1,95 @@
+import { NotFoundError } from '../errors'
```

Comment:
do we need consent model?

Reply:
relevant to https://github.com/mojaloop/als-consent-oracle/pull/3#discussion_r560924716